app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/AccessFactorSourceProxyContract.kt

```diff
@@ -98,19 +99,19 @@ sealed interface AccessFactorSourcesInput {
 
         val factorSource: FactorSource
         val isForLegacyOlympia: Boolean
-        val nextDerivationPathOffset: UInt // is used as pointer when user clicks "scan the next 50"
+        val nextDerivationPathIndex: HdPathComponent // is used as pointer when user clicks "scan the next 50"
```

Comment:
Wrapped the value with `HdPathComponent`. A simple `UInt` has no meaning with no context. We need to know if this index is calculated in the local key space or in the global.